src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -79,4 +79,33 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+  
+  public List<Message> getAllMessages(){
+	  List<Message> messages = new ArrayList<>();
```

Comment:
Looks like you are using tab for indentation. You should use spaces only. You can use 2 or 4 spaces. To do that you need to set your editor.